v22.1/backup-and-restore-overview.md

```diff
@@ -49,6 +49,10 @@ The following table outlines SQL statements you can use to create, configure, pa
 [`RESUME SCHEDULES`](resume-schedules.html) | Resume paused backup schedules.
 [`DROP SCHEDULES`](drop-schedules.html) | Drop backup schedules.
 
+## {{ site.data.products.db }} backup types
+
+{% include cockroachcloud/ccloud/backup-types.md %} 
```

Comment:
Also, I think we should pull this section of CockroachDB Backup types to the very beginning of this page. So the order would be:

Types of Backups supported by CockroachDB Cloud
[Backup and restore types](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#backup-and-restore-types) and their product availability
[SQL statements](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#backup-and-restore-sql-statements) for working with backups and restores
[Storage](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#backup-storage) for backups

Reply:
changed the order of the page, will work with Kathryn for updating the table and its respective product availability

Reply:
Liv's right, this table needs a general overhaul. I've created an issue and will complete this separately as part of the December milestone: https://cockroachlabs.atlassian.net/browse/DOC-6283